scripts/db/schema.sql

```diff
@@ -66,25 +190,23 @@ CREATE TABLE assets (
     source character varying(255),
     thumbnail_url character varying(255),
     title character varying(255),
-    trending_percentile integer DEFAULT 0 NOT NULL,
-    trending_score integer DEFAULT 0 NOT NULL,
     type enum_assets_type NOT NULL,
     url character varying(255),
     views integer DEFAULT 0,
     visible boolean DEFAULT true NOT NULL,
     created_at TIMESTAMP WITH TIME ZONE NOT NULL,
     updated_at TIMESTAMP WITH TIME ZONE NOT NULL
 );
-ALTER TABLE assets OWNER TO squiggy;
```

Comment:
I'm experimenting to see if we can get away without using these hardcoded ownership statements in the schema. The actual app user might be called "squiggy" or "app_squiggy" depending on environment.